lib/chromedriver.js

```diff
@@ -498,22 +498,24 @@ class Chromedriver extends events.EventEmitter {
   }
 
   /**
-   *
+   * @param {string} [cdName]
    * @param {string} [cdVersion]
    */
-  syncProtocol(cdVersion) {
+  syncProtocol(cdName, cdVersion) {
```

Comment:
I think we can go without the additional argument here as it is only needed for logging. We can simple say "The driver"